04-kogito-travel-agency/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,669 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Kogito Travel Agency</title>
+
+    <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+    <link rel="stylesheet" href="http://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+</head>
+<body>
+<div class="container" role="main">
+    <div class="jumbotron">
+        <h1>Kogito Travel Agency</h1>
+
+        <p>Plan your next trip with Kogito Travel Agency, it's easy and fast!</p>
+    </div>
+    <div class="page-header">
+        <div class="btn-group pull-right" role="group" style="padding-top: 10px;">
+            <button class="btn btn-primary" onclick="load();">
+                <span class="glyphicon glyphicon-refresh"></span> Refresh
+            </button>
+            <button class="btn btn-primary product-add" data-action="add" data-toggle="modal"
+                    data-target="#productModal">
+                <span class="glyphicon glyphicon-plus"></span> Plan new trip
+            </button>
+        </div>
+        <h1>Travels</h1>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            
+            <table class="table">
+                <thead>
+                <tr>
+                    <th>Traveller name</th>
+                    <th>Destination</th>
+                    <th>Dates</th>
+                    <th>Visa required</th>
+                    <th>Hotel</th>
+                    <th>Flight</th>
+                    <th>Actions</th>
+                </tr>
+                </thead>
+                <tbody id="content">
+                <!-- filled using Ajax -->
+                </tbody>
+            </table>
+        </div>
+    </div>
+</div>
+</div>
+<script>
+    var graphql = "http://localhost:8180/graphql";
```

Comment:
same here, wouldn't it be enough to use `/graphql` and then it will resolve to the host and protocol based on the location this file was downloaded from?